src/mo_frontend/lexer_lib.ml

```diff
@@ -5,13 +5,16 @@
 *)
 type mode = {
   privileged : bool;
+  verification : bool;
 }
 
 let mode : mode = {
   privileged = Option.is_some (Sys.getenv_opt "MOC_UNLOCK_PRIM");
+  verification = Option.is_some (Sys.getenv_opt "MOC_UNLOCK_VERIFICATION");
```

Comment:
Do we need the environment variable at all? Why not just use flags.viper (or add one if not present), implicitly set by --viper?

Reply:
The problem with that is that it won't compile the `make -C test/viper` mode.

Reply:
See 571b576ca